simplepath/utils.py

```diff
@@ -0,0 +1,35 @@
+"""Module that describes all utility functions for Simplepath."""
+
+
+def convert_object_to_dict(data_object):
+    """Convert all of the non-function attributes of an object into
+       a dictionary.
+    Args:
+        data_object(object): any Python object
+    Returns:
+        The dictionary representation of all fields in the Python object.
+    """
+
+    if hasattr(data_object, "__dict__"):
```

Comment:
in the rest of the lib, I use `'` as string quote character...

also this does not handle all objects. this will not consider objects with `__slots__`:

```python
if hasattr(data_object, '__dict__') or hasattr(data_object, '__slots__'):
     if hasattr(data_object, '__slots__'):
          object_dict = {k: getattr(data_object, k) for k in data_object.__slots__}
     else:
          object_dict = vars(data_object)
```